website/docs/components/icon.mdx

```diff
@@ -11,11 +11,12 @@ import { Grid, Flex, useTheme, Icon, Text } from "@chakra-ui/core"
 
 # Icon
 
-The icon component renders `svg` icons. Chakra UI provides basic interface
-icons. To add custom icons, [read the guide](#adding-custom-icons).
+By default, Chakra UI comes with a [set of basic interface icons](#all-icons).
+`Icon` component renders `svg` element and can be styled using `themeKey:`
+`"Icon"`. To add any custom icons, [follow this guide](#adding-custom-icons).
```

Comment:
```suggestion
`Icon` component renders `svg` element and can be styled using `themeKey:`
`"Icon"`.
```

Thought we should add the default `themeKey` to each of the component somewhere.